src/components/home/info.tsx

```
@@ -20,11 +33,30 @@ export default function MainInfo() {
+                                <li
+                                    className="whitespace-nowrap text-ellipsis overflow-hidden"
+                                    key={notice.noticeId}
+                                    onClick={() => navigate(`/notice/${notice.noticeId}`)}
+                                    onKeyDown={(e) => {
+                                        if (e.key === 'Enter' || e.key === ' ') {
+                                            navigate(`/notice/${notice.noticeId}`);
+                                        }
+                                    }}
+                                    tabIndex={0}
+                                    role="button"
+                                >
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)